### PR TITLE
user-docs: add FDO and Simplified Installer documentation

### PR DESCRIPTION
**Pull request by mmartinv**
Document how to use FDO with Simplified installer to
perform FIDO device Onboarding.

Resolves: fedora-iot/iot-distro#30

### CODE REVIEW

File: user-docs/modules/ROOT/pages/fdo-terminology.adoc [hunk 1]
```diff
@@ -0,0 +1,43 @@
+= FIDO device onboarding terminology
+
+Learn more about the FIDO device onboarding terminology.
+
+.FDO terminology
+
+|===
+|Commands | Description
```
Comment (nullr0ute) [inline user-docs/modules/ROOT/pages/fdo-terminology.adoc:8]: Is it a command? It seems more a term given the above section header

File: user-docs/modules/ROOT/pages/fdo-terminology.adoc [hunk 2]
```diff
@@ -0,0 +1,43 @@
+= FIDO device onboarding terminology
```
Comment (nullr0ute) [inline user-docs/modules/ROOT/pages/fdo-terminology.adoc:1]: Is glossary a better word than terminology?

File: user-docs/modules/ROOT/pages/fdo-terminology.adoc [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+Contains the following information:
+
+* Owner (`fdo-owner-onboarding-service`)
+
+* Rendezvous Server - FIDO server (`fdo-rendezvous-server`)
+
+* Device (at least one combination)  (`fdo-manufacturing-service`)
+
+| Device Credential (DC) | Key credential and rendezvous stored in the device at manufacture.
```
Comment (nullr0ute) [inline user-docs/modules/ROOT/pages/fdo-terminology.adoc:26]: Should they be in alphabetical order? Also I think we're missing rendezvous definitions.

**Review by nullr0ute — COMMENTED** [review]
A few minors bits

File: user-docs/modules/ROOT/pages/fdo-terminology.adoc (hunk 1 above)
Comment (mmartinv) [reply user-docs/modules/ROOT/pages/fdo-terminology.adoc:8]: Honestly, this document was copied from the downstream documentation and I haven't reviewed it :(

Comment (mmartinv) [reply user-docs/modules/ROOT/pages/fdo-terminology.adoc]: Changed

File: user-docs/modules/ROOT/pages/fdo-terminology.adoc (hunk 2 above)
Comment (mmartinv) [reply user-docs/modules/ROOT/pages/fdo-terminology.adoc]: Changed

File: user-docs/modules/ROOT/pages/fdo-terminology.adoc (hunk 3 above)
Comment (mmartinv) [reply user-docs/modules/ROOT/pages/fdo-terminology.adoc]: Done

**Pull request merged**